cmd/car/car.go

```diff
@@ -79,7 +81,8 @@ func main() {
 
 	err := app.Run(os.Args)
 	if err != nil {
-		log.Fatal(err)
-		os.Exit(1)
+		log.Println(err)
+		return 1
```

Comment:
Would it make sense to utilise urfave/cli machinery like [`OsExiter`](https://pkg.go.dev/github.com/urfave/cli/v2#OsExiter) for handling exit codes here? Tests can then override that.

Reply:
I guess we technically could, but I'm not sure that we should. `func() int` feels much easier than abusing a library's global callback variable :)